match/lib/match/runner.rb

```diff
@@ -236,10 +250,16 @@ def fetch_provisioning_profile(params: nil, certificate_id: nil, app_identifier:
         self.files_to_commit << profile
       end
 
-      installed_profile = FastlaneCore::ProvisioningProfile.install(profile, keychain_path)
+      if Helper.mac?
+        installed_profile = FastlaneCore::ProvisioningProfile.install(profile, keychain_path)
+      end
       parsed = FastlaneCore::ProvisioningProfile.parse(profile, keychain_path)
       uuid = parsed["UUID"]
 
+      if params[:export_path]
+        FileUtils.cp(profile, params[:export_path])
```

Comment:
Do we want to do a `FileUtils.mkdir_p(params[:export_path])` here? There is a good chance a user might put a directory that doesn't exist 🙃 Either this or we need to verify on the `ConfigItem` that the path exists.

Reply:
Do we have any similar options for other actions where we could "steal" the behavior for consistency? I actually forgot to finish the `ConfigItem` here to be honest, because I couldn't think of where to look for how this could/should work.

Reply:
`cert`, `pem` and `sigh` all have an `output_path` option, so I renamed this one here as well. I also copied how those other actions create the directory at the beginning of `run` (only this time only if the param is supplied, as for the others it has a default value).